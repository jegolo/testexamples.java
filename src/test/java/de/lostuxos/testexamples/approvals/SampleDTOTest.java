package de.lostuxos.testexamples.approvals;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

public class SampleDTOTest {

    private static class Person {
       private String name;
       private String firstname;
       private String age;

        public String getName() {
            return name;
        }

        public void setName(String name) {
            this.name = name;
        }

        public String getFirstname() {
            return firstname;
        }

        public void setFirstname(String firstname) {
            this.firstname = firstname;
        }

        public Person(String name, String firstname, String age) {
            this.name = name;
            this.firstname = firstname;
            this.age = age;
        }

        public String getAge() {
            return age;
        }

        public void setAge(String age) {
            this.age = age;
        }

        @Override
        public String toString() {
            return "Person{" +
                    "name='" + name + '\'' +
                    ", firstname='" + firstname + '\'' +
                    ", age='" + age + '\'' +
                    '}';
        }
    }

    @Test
    public void testDto() {
        Person person = new Person("Mustermann", "Max", "15");

        Approvals.verify(person);
    }
}
