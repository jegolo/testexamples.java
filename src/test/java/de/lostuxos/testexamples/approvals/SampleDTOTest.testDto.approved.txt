Person{name='Mustermann', firstname='Max', age='15'}